integration-tests/src/test/java/tachyon/client/TachyonFileSystemIntegrationTest.java

```diff
@@ -156,7 +157,8 @@ public void mkdirTest() throws IOException, TachyonException {
     MkdirOptions options = new MkdirOptions.Builder(new TachyonConf()).setRecursive(true).build();
     for (int k = 0; k < 10; k ++) {
       Assert.assertTrue(sTfs.mkdir(new TachyonURI(uniqPath + k), options));
-      Assert.assertTrue(sTfs.mkdir(new TachyonURI(uniqPath + k), options));
+      mThrown.expect(FileAlreadyExistsException.class);
```

Comment:
after `mThrown.expect`, the unit test case will simply and will not continue the loop.